package.json

```diff
@@ -38,8 +38,7 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@stryker-mutator/core": "^3.3.1",
-    "@stryker-mutator/javascript-mutator": "^3.3.1",
+    "@stryker-mutator/core": "^4.3.1",
```

Comment:
FYI I think this is slightly outdated now. If we merge this PR, Renovate *should* be able to make the update for us.